### DISCUSSION TITLE
Markdown viewer

### DISCUSSION THREAD

**Discussion by AndyGUK** [Budibase/budibase#4693] [General]
I've just discovered the markdown viewer. I've spent weeks building a CRUD for our company and only just discovered it... I don't seem to be able to find anything on the docs site about it. Is there any more info on this such as the limitations etc?

**Comment by shogunpurple** [comment]
Hi @AndyGUK - it's a new feature that was only shipped over the past week or two. It was announced in the release discussion here:
https://github.com/Budibase/budibase/discussions/4527

In terms of limitations, is there anything specific you had in mind?

**Reply by AndyGUK** [reply]
Ah, I see, that would be why! It would be good to know if it's possible to use bootstrap in an element built with the markdown viewer. I did skim the release but I was expecting to see something like 'HTML embed'. My own fault really. Thanks for clarifying.

Such a game changer having this available now!
